src/main/resources/db_scripts/db_script_v5.sql

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2018 ABSA Group Limited
```

Comment:
Are you going to merge v5 and v6 scripts ?

Reply:
No, they are not the same. The plan is to apply v4_to_v5 first, and v5_to_v6 later, e.g. with the next deployment, to enable simple rollback in case something's wrong.